extensions/wikia/Chat2/NodeApiClient.class.php

```diff
@@ -110,7 +127,9 @@ static private function makeRequest($params){
 		global $wgReadOnly;
 		wfProfileIn( __METHOD__ );
 		$response = "";
-		
+
+		ChatHelper::debug( __METHOD__ . ': Method called ', $params );
```

Comment:
This is the only place we're doing a `debug` call, all the others are `info`. Is there a reason to have this distinction? Can we just make this an `info` call and get rid of that extra method since it's only being used here?